kotlin/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM iron/java
+
+ENV VERSION 1.0.5
+
+ENV KOTLIN_HOME=/usr/share/kotlin
+ENV KOTLIN_COMPILER_URL https://github.com/JetBrains/kotlin/releases/download/v${VERSION}/kotlin-compiler-${VERSION}.zip
```

Comment:
Please put all 3 env vars on the same `ENV` line to reduce layers. You can use the `=` form to achieve this.